.github/workflows/test_ruby.yml

```diff
@@ -102,23 +112,26 @@ jobs:
         with:
           credentials: ${{ secrets.GAR_SERVICE_ACCOUNT }}
           bazel-cache: ruby_macos/${{ matrix.version }}
-          bazel: test //ruby/... --test_env=KOKORO_RUBY_VERSION=${{ matrix.version }}
+          bazel: test //ruby/... --test_env=KOKORO_RUBY_VERSION=${{ matrix.version }} --test_env=BAZEL=true ${{ matrix.ffi == 'FFI' && '--//ruby:ffi=enabled --test_env=PROTOCOL_BUFFERS_RUBY_IMPLEMENTATION=FFI' || '' }}
```

Comment:
Same comment here, you could add a `flags` and a `suffix` argument to capture this and make it more readable

Reply:
I'm not sure I'm visualizing the same change you have in your mind - can you give an example? From a readability (and a reducing opportunity to introduce a typo) standpoint, I wouldn't want to repeat the same value of the `flags` attribute for every config that needs it.